### PR TITLE
Add demo example in the quickstart guide

### PR DESCRIPTION
**Pull request by jlevesy**
## What does this PR do ? 

This PR updates the quickstart guide to add a `using maesh` section, demonstrating how to use maesh.

Fixes: #323 

## How to test this

Check netlify deployment and run this quickstart guide.

### CODE REVIEW

File: docs/content/quickstart.md [hunk 1]
```diff
@@ -38,6 +30,174 @@ helm installation to perform all steps it needs is to edit the
 Assuming `tiller` is deployed in your `kube-system` namespace, this will
 give it very open permissions.
 
-### Installation Examples
+## Installing Maesh
+
+```bash tab="Command"
```
Comment (Ullaakut) [inline docs/content/quickstart.md]: If there's a command tab, I guess there should be an expected output as well?

File: docs/content/quickstart.md [hunk 2]
```diff
@@ -38,6 +30,174 @@ helm installation to perform all steps it needs is to edit the
...
+helm repo add maesh https://containous.github.io/maesh/charts
+helm repo update
+helm install --name=maesh --namespace=maesh maesh/maesh
+```
+
+## Using Maesh
+
+Let's deploy a server application and a client application under the `maesh-test` namespace.
```
Comment (Ullaakut) [inline docs/content/quickstart.md]: Maybe we should make it more obvious that this is an example and not the way they have to use Maesh 😅

```suggestion
As an example, let's deploy a server application and a client application under the `maesh-test` namespace.
```

File: docs/content/quickstart.md [hunk 3]
```diff
@@ -38,6 +30,174 @@ helm installation to perform all steps it needs is to edit the
...
+IP: 10.42.2.10
+IP: fe80::a4ec:77ff:fe37:1cdd
+RemoteAddr: 10.42.2.9:46078
+GET / HTTP/1.1
+Host: server.maesh-test.svc.cluster.local
+User-Agent: curl/7.64.0
+Accept: */*
+```
+
+You can note that all this server application is doing is to answer back the content of the request it receives.
```
Comment (Ullaakut) [inline docs/content/quickstart.md]:
```suggestion
You can note that all this server application is doing is to respond with the content of the request it receives.
```

File: docs/content/quickstart.md [hunk 4]
```diff
@@ -38,6 +30,174 @@ helm installation to perform all steps it needs is to edit the
...
+Uber-Trace-Id: 3f9e7129a059f70:7e889a1ebcb147ac:3f9e7129a059f70:1
+X-Forwarded-For: 10.42.2.9
+X-Forwarded-Host: server.maesh-test.maesh
+X-Forwarded-Port: 80
+X-Forwarded-Proto: http
+X-Forwarded-Server: maesh-mesh-w95q2
+X-Real-Ip: 10.42.2.9
+```
+
+Note the presence of `X-Forwarded` headers as well than other instrumentation headers like `Uber-Trace-Id`, indicating than your request has been processed and instrumented by Maesh.
```
Comment (Ullaakut) [inline docs/content/quickstart.md]:
```suggestion
Note the presence of `X-Forwarded` headers as well as other instrumentation headers like `Uber-Trace-Id`, indicating than your request has been processed and instrumented by Maesh.
```

File: docs/content/quickstart.md (hunk 1 above)
Comment (jlevesy) [reply docs/content/quickstart.md]: Way too long, isn't it ?

Comment (dtomcej) [reply docs/content/quickstart.md]: You could also do a `[...]` truncated output, just to show the top and bottom perhaps...

Comment (jlevesy) [reply docs/content/quickstart.md]: Yep okay.

**Pull request merged**